src/main/java/models/SpanishGame.java

```diff
@@ -0,0 +1,31 @@
+package models;
+
+import java.util.ArrayList;
+
+/**
+ * Created by harderg on 2/14/16.
+ */
+public class SpanishGame extends Game {
+
+    /*public void buildDeck() {
+        for(int i = 1; i < 14; i++){
+            if (i < 8 || i > 10) {
+                deck.add(new Card(i, SpanishSuit.Clubs));
+                deck.add(new Card(i, SpanishSuit.Coins));
+                deck.add(new Card(i, SpanishSuit.Swords));
+                deck.add(new Card(i, SpanishSuit.Cups));
+            }
+        }
+    }*/
+
+    public void buildDeck() {
+        for(int i = 1; i < 14; i++){
+            if (i < 8 || i > 10) {
```

Comment:
I am trying to understand the point of the nested if statement. What does it actually do?